### PR TITLE
fix issue#12. Give some slack for etcd to come alive.

### PR DESCRIPTION
**Pull request by rajatchopra**
@mrunalp 
Review please.

### CODE REVIEW

**Comment by rajatchopra** [comment]
Fixes #12

File: main.go [hunk 1]
```diff
@@ -118,6 +118,7 @@ func main() {
 	} else if opts.master {
 		err := be.StartMaster(opts.sync)
 		if err != nil {
+			log.Errorf(err.Error())
```
Comment (mrunalp) [inline main.go]: I think you can just use err here.

Comment (rajatchopra) [inline main.go]: Fixed. Errorf requires a string, but we do not use any formatting here.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**